tests/test_isrTaskLSST.py

```diff
@@ -476,6 +477,38 @@ def test_isrFlat(self):
         ratio = result2.exposure.image.array / result.exposure.image.array
         self.assertFloatsAlmostEqual(ratio, flat_nodefects.image.array, atol=1e-4)
 
+    def test_isrBrighterFatter(self):
+        """Test processing of a flat frame."""
+        mock_config = self.get_mock_config_clean()
+        mock_config.doAddSource = True
+        mock_config.sourceFlux = [75000.0]
+        mock_config.isTrimmed = False
+        mock_config.doAddBrighterFatter = True
+
+        mock = isrMockLSST.IsrMockLSST(config=mock_config)
+        input_exp = mock.run()
+
+        isr_config = self.get_isr_config_minimal_corrections()
+        isr_config.doBrighterFatter = True
+        isr_config.doApplyGains = True
+
+        isr_task = IsrTaskLSST(config=isr_config)
+        with self.assertNoLogs(level=logging.WARNING):
+            result = isr_task.run(
+                input_exp.clone(),
+                bfKernel=self.bf_kernel,
+            )
+
+        # Measure the source size in the BF-corrected image.
+        # The injected source is a Gaussian with 3.0px
+        image = galsim.ImageF(result.exposure.image.array)
+        hsm_result = galsim.hsm.FindAdaptiveMom(image, guess_centroid=galsim.PositionD(50., 25.),
+                                                strict=False)
+        measured_sigma = hsm_result.moments_sigma
+
+        # With the BF correction, the flats should have the same variation.
+        self.assertFloatsAlmostEqual(measured_sigma, 3.0, rtol=1.0e-3)
+
```

Comment:
If we add in the sky and everything else (which we should on this or a second test) we should also check the equality everywhere except around the source (how far out is up to you/the tests).  It'll also be important to add in the defects, etc, to make sure that all the interpolation code is working correction in the BF context.